ingest/src/sources/mongodb-university/MongoDbUniversityDataApiClient.ts

```diff
@@ -0,0 +1,179 @@
+import fetch from "node-fetch";
+/**
+  Item in the Thought Industries catalog as returned from the
+  MongoDB University Data API.
+ */
+export interface TiCatalogItem {
+  id: string;
+  ti_id: string;
+  /**
+        IDs for videos associated with the catalog item.
+       */
```

Comment:
Weird comment indentation in this file can we fix?

Reply:
think that's some copy-pasta. will fix